crates/astria-conductor/src/executor.rs

```diff
@@ -122,7 +120,9 @@ struct Executor<C> {
     /// we need to track the mapping of sequencer block hash -> execution block hash
     /// so that we can mark the block as final on the execution layer when
     /// we receive a finalized sequencer block.
-    sequencer_hash_to_execution_hash: HashMap<Base64String, Vec<u8>>,
+    ///
+    /// TODO: change Vec<u8> execution hash to its own ExecutionHash type
```

Comment:
I added this; this would basically just be making a type alias `type ExecutionHash = Vec<u8>`. I don't think `ExecutionHash` should be `tendermint::Hash` as the hash format/algorithm may differ for execution environments